### ISSUE TITLE
Bypass mixer in exclusive mode

### ISSUE THREAD

**Issue by roberth1990** [PipeWire/pipewire#126]
Hello

Is there any plans implement a mode similar to configure an application to send audio to alsa device "hw:x,y" to bypass mixing, resampling etc and the application takes exclusive control over the device?

Sorry if it's a poor explanation.

**Comment by v-fox** [comment]
I hope not. Biggest advantage of such multimedia hardware manager is to not let any one application to screw up things for entire user session or even system. It should run all hardware at maximum of its capabilities, mix data at maximum practical precision (offload processing to DSPs when possible), handle all conversion and processing-node management.

Getting rid of ALSA's hardcoded silly device identifiers (numeric and textual, that are not unique or constant) based solely on driver loading order is also would be nice. That's one thing that PA did right.

**Comment by wtay** [comment]
Yes. There is a possibility to get exclusive access to the device. This is all managed by the session manager now so it has to decide if this is something that is allowed for the requesting app etc..  PipeWire will attempt to assign a constant unique UUID per device (not implemented yet), regardless of what alsa device it is/was.
